documentation/book/con-cluster-operator.adoc

```diff
@@ -0,0 +1,21 @@
+[id='cluster-operator-{context}']
+= Overview of Cluster Operator
+
+The Cluster Operator is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble. As part of the Kafka cluster, it can also deploy the Topic Operator which provides operator-style topic management via ConfigMaps. 
+
+ifdef::Kubernetes[]
```

Comment:
This is fully supported on both Kubernetes and OpenShift. I don't think we need to have the `ifdef` here.

Reply:
@scholzj yes we do because the Kubernetes info won't be included in the product docs.